README.md

```diff
@@ -111,34 +86,33 @@ After you're finished please remove all the comments and instructions!
 
 > Add a link to your deployed project.
 
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
+- [Live Demo Link](http://127.0.0.1:5500/index.html))
```

Comment:
- [ ] This link might only work on your system when your liver server is running because you have the source file opened on your local machine but we don't have access to it. So, since you have not deployed this on any server with global access, I suggest you remove the link, and let people know that the link is not available. What do you think?